neo4j/src/test/java/org/hibernate/ogm/datastore/neo4j/test/EmbeddedGraphDatabaseFactoryTest.java

```diff
@@ -68,6 +66,51 @@ public void testLoadMalformedPropertiesLocation() throws Exception {
 		factory.create().shutdown();
 	}
 
+	@Test
+	public void testInstantiationTwoFactoriesForOnePath() {
+		//create first factory
+		EmbeddedNeo4jGraphDatabaseFactory factory1 = new EmbeddedNeo4jGraphDatabaseFactory();
+		Properties properties = new Properties();
+		properties.put( Neo4jProperties.DATABASE_PATH, dbLocation );
+		properties.put( Neo4jProperties.CONFIGURATION_RESOURCE_NAME, neo4jPropertiesUrl().toExternalForm() );
+		factory1.initialize( properties );
+		GraphDatabaseService graphDatabaseService1 = factory1.create();
+
+		//create second factory
+		EmbeddedNeo4jGraphDatabaseFactory factory2 = new EmbeddedNeo4jGraphDatabaseFactory();
+		factory2.initialize( properties );
+		GraphDatabaseService graphDatabaseService2 = factory2.create();
+
+		assertTrue( "Two instances must be same!", graphDatabaseService1 == graphDatabaseService2 );
+
+		graphDatabaseService1.shutdown();
+		graphDatabaseService2.shutdown();
+	}
+
```

Comment:
When you mention that it was not possible to have different paths I didn't understand you meant two factories using the same folder for the same db.

Using the same folder in Neo4j embedded is not possible and the fact the there is an error when one tries to is correct. Also in this case if I want to use two different set of properties but the path is the same only the irst set of properties is accepted.

And what happens if I want to shutdown a factory but not the other?